shipping/uc_quote/js/uc_quote.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Utility functions to display settings summaries on vertical tabs.
+ */
+
+(function ($) {
+
+Backdrop.behaviors.ucQuoteFieldsetSummaries = {
+  attach: function (context) {
+    $('fieldset.product-shipping', context).backdropSetSummary(function(context) {
+      var vals = [];
+
+      if ($('#edit-shippable', context).is(':checked')) {
+        vals.push(Backdrop.t('Shippable'));
+      }
+      else {
+        vals.push(Backdrop.t('Not shippable'));
+      }
+
+      var shipping_type = $('#edit-shipping-type option[selected]').text();
```

Comment:
The correct jquery syntax is:

```suggestion
      var shipping_type = $('#edit-shipping-type option:selected').text();
```